test/component/deceased/testAlias.js

```diff
@@ -20,7 +23,18 @@ describe('deceased-alias', () => {
     describe('Verify Content, Errors and Redirection', () => {
 
         it('test right content loaded on the page', (done) => {
-            testWrapper.testContent(done, []);
+            set(sessionData, 'deceased.firstName', 'John');
+            set(sessionData, 'deceased.lastName', 'Doe');
```

Comment:
We can set the `sessionData` object directly here instead of using `set`, for example:

```
const sessionData = {
  deceased: {
    firstName: 'John',
    surname: 'Doe'
  }
}
```

As we're setting the entire object we don't need any of the convenience `set` gives us and therefore we don't need to require an additional library.